### ISSUE TITLE
Blender Launcher opens but won't launch any Blender Builds

### ISSUE THREAD

**Issue by hopeinformer** [DotBow/Blender-Launcher#24]
This is a Blender Launcher but it isn't launching any of the Blender Builds I have downloaded. It’ll download them into the  directory I specify but it won’t launch. When I click the “launch” button there will be a quick flash if a dark window for a quick fraction of a second then a number 1 on the build version for a moment, then the 1 disappears. I have Blender 2.90.1 already installed on my laptop but during the initial setup I specified the directory where it was installed but it won't work. So I downloaded them from the launcher thinking that was necessary. Another issue I noticed is when I right-click on a build version and select “show in folder” it’ll only open the folder in Visual Studio Code.

**To Reproduce**
1. Download the build version. I’ve chosen the 2 latest stable versions; 2.90.1 & 2.83.7
2. Click the desired version. I’ve tried them both. I have also set the 2.90.1 version as my favorite and tried to use the launcher from the upper toolbar panel
3. Nothing happens. 

**Expected behavior**
I expected to have this program launch Blender

**Screenshots**
Added.

**System (please complete the following information):**
 - Operation system: Linux, Ubuntu 20.04 LTS 64bit 
 - Blender Launcher version: [v1.5.0]
 - Log file: attached

**Additional context**
I have tried to download the launcher again and use the different file in different locations. I have tried to add different builds. Nothing seems to work. That’s about it.

[blender launcher gui](https://user-images.githubusercontent.com/52873265/95656666-b6b33e00-0ac4-11eb-888c-30d8fea52b04.png)
![blender launcher gui with settings window](https://user-images.githubusercontent.com/52873265/95656669-badf5b80-0ac4-11eb-8f70-4835bb5efefd.png)
![blender launcher gui launch pressed](https://user-images.githubusercontent.com/52873265/95656727-245f6a00-0ac5-11eb-8945-3cc6cae6e61c.png)

[Blender Launcher.log](https://github.com/DotBow/Blender-Launcher/files/5358438/Blender.Launcher.log)

**Comment by DotBow** [comment]
Hi!
Sorry for experiencing such issue 😞 It seems to be related to #21.
I will rebuild BL asap using different environment and send you build to test it.

**Comment by hopeinformer** [comment]
Thank you very much DotBow.

**Comment by DotBow** [comment]
Hi!
Please, check this build:
https://drive.google.com/file/d/1udvGLdbqpQYLBFUyR7Tyl1UrDNdUE7A7/view?usp=sharing

**Comment by hopeinformer** [comment]
Thank you for the quick help. Unfortunately its the same.
That's alright. It's a great app for downloading the newest builds. I can easily enough launch the builds from the folder.

**Comment by DotBow** [comment]
Nuh, I feel bad that you have troubles using it 😞 
There is another one, please note to exit all instance of BL before opening this one:
https://drive.google.com/file/d/137HXABteclFDr0K9Lb6viZZnwxq2ycBC/view?usp=sharing

**Comment by hopeinformer** [comment]
I'm still getting the same issue. I'm going to download a clean unaltered version of Ubuntu 20.04 as well as some other distros and run them in a virtual machine to see if it works. It could be something in my OS's settings that's causing this. I'll get back to you after the experiment.

**Comment by hopeinformer** [comment]
I have downloaded and installed a clean version of Ubuntu 20.04 in a virtual machine and there's still the same problem. I tried to run it in Fedora but wasn't able to yet. Still working out some bugs with getting it to connect to the internet.

**Comment by DotBow** [comment]
> I have downloaded and installed a clean version of Ubuntu 20.04 in a virtual machine and there's still the same problem. I tried to run it in Fedora but wasn't able to yet. Still working out some bugs with getting it to connect to the internet.

It doesn't seem to have any sense - I used to build under clean installation of Ubuntu 18 lately.
What internet connection issue you are talking about?

**Comment by DotBow** [comment]
Ok, this one is 100% works on Ubuntu 20, however I still don't have a clue why 😕 
As far as I can tell it might be some issues with PyInstaller or Python 3.8 differences (it bundles with latest Linux).

https://drive.google.com/file/d/189cJvBSHxg8c9_lXtpuj2ZAwoYmDKLY4/view?usp=sharing

**Comment by manujarvinen** [comment]
> Ok, this one is 100% works on Ubuntu 20, however I still don't have a clue why confused
> As far as I can tell it might be some issues with PyInstaller or Python 3.8 differences (it bundles with latest Linux).
> 
> https://drive.google.com/file/d/189cJvBSHxg8c9_lXtpuj2ZAwoYmDKLY4/view?usp=sharing

Doesn't work on my Xubuntu 19.10, as usual, dang it

**Comment by hopeinformer** [comment]
Whoa! You totally did it! This works! I'm going to buy you a beer. Thank you so much for your commitment in getting this to work. I appreciate it greatly.

**Comment by hopeinformer** [comment]
>
> Doesn't work on my Xubuntu 19.10, as usual, dang it
That's strange it didn't work on Xbuntu. That distro is the same as the Ubuntu just with a different desktop environment. Which can be added to the Ubuntu by running the command [sudo apt-get install xubuntu-desktop]. I tried it but like the Unity more than the XFCE.
Although, it could be that you're using the older build. The 20.04 is the LTS

**Comment by DotBow** [comment]
> 
> 
> > Ok, this one is 100% works on Ubuntu 20, however I still don't have a clue why confused
> > As far as I can tell it might be some issues with PyInstaller or Python 3.8 differences (it bundles with latest Linux).
> > https://drive.google.com/file/d/189cJvBSHxg8c9_lXtpuj2ZAwoYmDKLY4/view?usp=sharing
> 
> Doesn't work on my Xubuntu 19.10, as usual, dang it

Yeah, that was the point - something goes wrong with compatibility.
Will try to go deeper with what's going wrong.
For the time being you can try build BL by your self following [Development](https://github.com/DotBow/Blender-Launcher/wiki/Development) page while I'm searching for better solution.

**Comment by manujarvinen** [comment]
> For the time being you can try build BL by your self following [Development](https://github.com/DotBow/Blender-Launcher/wiki/Development) page while I'm searching for better solution.

I got this error while running the ```$ pipenv run start```

![pipenv_run_start](https://user-images.githubusercontent.com/5437697/96376812-c2b89300-1189-11eb-8e08-d9006ecfd4b2.png)

Would it be because my python3-dev claims to be up to date, but seems to be 3.7.5 ?

![python3-dev](https://user-images.githubusercontent.com/5437697/96376779-9a309900-1189-11eb-82ff-97f6614eae76.png)

I installed python-3.8 before starting everything, though

**Comment by manujarvinen** [comment]
Okay, now that my Xubuntu is 20.04, the **Blender_Launcher_v1.6.0_Ubuntu_20_LTS.zip** works as it should - problem solved, in a way. (The official 1.6.0 Linux didn't work still even in this 20.04 Xubuntu)

**Comment by DotBow** [comment]
> 
> 
> Okay, now that my Xubuntu is 20.04, the **Blender_Launcher_v1.6.0_Ubuntu_20_LTS.zip** works as it should - problem solved, in a way. (The official 1.6.0 Linux didn't work still even in this 20.04 Xubuntu)

Oh, sorry for getting so much trouble :pray:
After couple of hours of additional research it seems I finally found out what was wrong - I needed to be set manually a proper library path before using specific packages. Now everything works on other Linux distros in my case.

Here is the builds with fix:
https://drive.google.com/file/d/1Qc7-b2PffBngiRdySLPs_su7AxhYB-fo/view?usp=sharing

**Comment by manujarvinen** [comment]
You're right! :) That also works as it should on my Xubuntu 20.04

**Comment by DARKFRACTURES** [comment]
> Here is the build with fix:
> https://drive.google.com/file/d/1Qc7-b2PffBngiRdySLPs_su7AxhYB-fo/view?usp=sharing

Had the same issue on pop os 20.04, this build fixed it!

**Comment by hopeinformer** [comment]
Thank you DotBow for your help. It has been working flawlessly.
